Source/Aptabase/Private/AptabaseAnalyticsProvider.cpp

```diff
@@ -47,7 +95,14 @@ bool FAptabaseAnalyticsProvider::SetSessionID(const FString& InSessionID)
 
 void FAptabaseAnalyticsProvider::FlushEvents()
 {
-	UE_LOG(LogAptabase, Log, TEXT("Aptabase implementation doesn't cache events"));
+	UE_LOG(LogAptabase, Verbose, TEXT("Flushing %s batched events."), *LexToString(BatchedEvents.Num()));
+
+	for (const auto& Event : BatchedEvents)
```

Comment:
There's a new endpoint `/api/v0/events` that accepts an array of Event to avoid having to send one by one.

There's just one added complexity that it currently only supports up to 25 events in a single call, so the array has to be chunked before that